microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -143,6 +146,7 @@ You also need to add a new REGKEY `<myApp.exe> = --remote-debugging-port=9222` u
 
    ![The resulting registry key in the Registry Editor](./debug-visual-studio-code-images/set-debugging-port-registry-key.png)
 
+> Instead of adding the above registry key, you can set the `WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS` environment variable to `--remote-debugging-port=9222`. Make sure that your application was started after the environment variable had been set, and that your application inherits the variable. See [WewbView2 browser flags](../concepts/webview-features-flags.md) for more information.
```

Comment:
```suggestion
```
reverting this addition, per [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3384/files#r1965590535) 
@mikosg @dhveerap 
 in `debug-visual-studio-code.md`